README.md

```diff
@@ -107,6 +109,10 @@ And additional requirements are in [**requirements.txt**](https://github.com/gar
   * Migrate the changes to the database `$ python manage.py migrate`
   * Create admin `$ python manage.py createsuperuser`
   * Run the server `$ python manage.py runserver`
+  
+#### 💡️ **Pro Tip:** 
+  * Always keep your master branch in sync with the main repository (by running `$ git pull upstream master` on your local master branch) 
+  * Always create a new branch before making any changes (`$ git checkout -b <new-branch-name>`), never ever make any changes directly on the master branch.
 
```

Comment:
Corrected the changes.

Reply:
Now, did it.

Reply:
You put the `.` at the wrong place. See where I asked you to do it.

Reply:
In line 114 ?

Reply:
Yeah